src/main/java/nustracker/logic/parser/FilterCommandParser.java

```diff
@@ -124,4 +126,17 @@ public String getTrimmedArgs(String args, Prefix prefix) throws ParseException {
         return years;
     }
 
+    public StudentId[] getStudentIds(String[] splitStudentIds) throws ParseException {
+        StudentId[] studentIds = new StudentId[splitStudentIds.length];
+
+        for (int i = 0; i < studentIds.length; i++) {
+            try {
+                studentIds[i] = ParserUtil.parseStudentId(splitStudentIds[i]);
+            } catch (ParseException e) {
+                throw e;
+            }
```

Comment:
is there a need to catch and throw the exception here since the method already throws a parse exception?

Reply:
You're right, I'll remove it